library/patient.inc

```diff
@@ -812,6 +814,15 @@ function getPatientPID($args)
 
     return $returnval;
 }
+function getPatientDoBonly($pid)
+{
+    if (empty($pid)) {
+        return "";
```

Comment:
If you don't have a valid `pid` we should probably return `false` and maybe log an error? We don't really handle `try/catches` very well but you could even `throw` an `InvalidPIDException()`. `try/catches` would be the best way to go.

Reply:
My bad for following bad practices in the program. I could put a little more effort into the code and do something more fancy that what I see. I will put some thought into it since this is not the end of this and there is more work to be done because there is more information that is supposed to be on this page than I have currently included.